app/controllers/todos_controller.rb

```diff
@@ -22,7 +26,8 @@ def create
 
 
   def destroy
-    @todo = Todo.find(params[:id])
+    # @todos = current_user.todos
+    @todo = Todo.where(user_id: current_user.id).find(params[:id])
```

Comment:
For readability, consider using `@todo = current_user.todos.find(params[:id])` here.